src/test/java/io/github/zlika/reproducible/ZipStripperTest.java

```diff
@@ -28,23 +28,25 @@
 {
     /**
      * Tests stripping on a reference JAR file.
-     * @throws IOException 
+     * 
+     * @throws IOException
+     *             in case of error on test file operations
```

Comment:
Useless changeset that only differs on white spaces.
That's why I hate the IDE's automatic file formatters.

Reply:
Apologies again :-)